agent/app/agent_capability.go

```diff
@@ -311,12 +311,12 @@ func (agent *ecsAgent) capabilities() ([]*ecs.Attribute, error) {
 
 func (agent *ecsAgent) appendDockerDependentCapabilities(capabilities []*ecs.Attribute,
 	supportedVersions map[dockerclient.DockerVersion]bool) []*ecs.Attribute {
-	if _, ok := supportedVersions[dockerclient.Version_1_19]; ok {
+	if _, ok := supportedVersions[dockerclient.MinDockerAPIVersion]; ok {
 		capabilities = appendNameOnlyAttribute(capabilities, capabilityPrefix+"ecr-auth")
 		capabilities = appendNameOnlyAttribute(capabilities, attributePrefix+"execution-role-ecr-pull")
 	}
 
-	if _, ok := supportedVersions[dockerclient.Version_1_24]; ok && !agent.cfg.DisableDockerHealthCheck.Enabled() {
```

Comment:
the `supprtedVersions` map doesn't contain any versions earlier than the minimum version, so if we bump the minimum version greater than 1.24, then this map will no contain 1.24, and it will stop supporting the capability